Database/ReleaseScript/0633 -Add DNRUplanRegionCoordinator to DNRUplandRegion table.sql

```diff
@@ -0,0 +1,4 @@
+alter table dbo.DNRUplandRegion
+add DNRUplandRegionCoordinator int null constraint FK_DNRUplandRegion_Person_DNRUplandRegionCoordinator_PersonID foreign key (DNRUplandRegionCoordinator) 

```

Comment:
We should stick to the naming convention of ending all IDs with ID! Please rename to include ID.